components/api/db/queries.bal

```diff
@@ -32,17 +32,18 @@ public const string GET_ORG_COUNT_FOR_USER = "SELECT COUNT(ORG_NAME) FROM REGIST
 
 public const string GET_IMAGE_FOR_USER_FROM_IMAGE_NAME = "SELECT REGISTRY_ARTIFACT_IMAGE.ARTIFACT_IMAGE_ID,ORG_NAME,
                                     IMAGE_NAME,REGISTRY_ARTIFACT_IMAGE.SUMMARY, REGISTRY_ARTIFACT_IMAGE.DESCRIPTION,
-                                    REGISTRY_ARTIFACT_IMAGE.FIRST_AUTHOR, VISIBILITY, SUM(PUSH_COUNT), SUM(PULL_COUNT) FROM 
+                                    REGISTRY_ARTIFACT_IMAGE.FIRST_AUTHOR, VISIBILITY, SUM(PUSH_COUNT), SUM(PULL_COUNT), IFNULL((SELECT
```

Comment:
Do we need the IFNULL check ? Since the USER_ROLE is not nullable in the definition.

Reply:
We need it to handle requests with unavailable organization name or image name, Same applies to GET_ARTIFACT_FOR_USER_FROM_IMG_NAME_N_VERSION query